com.unity.render-pipelines.core/Runtime/Volume/Volume.cs

```diff
@@ -9,26 +9,39 @@ namespace UnityEngine.Rendering
     [CoreRPHelpURL("Volumes", "com.unity.render-pipelines.high-definition")]
     [ExecuteAlways]
     [AddComponentMenu("Miscellaneous/Volume")]
-    public class Volume : MonoBehaviour, IVolume
+    public partial class Volume : MonoBehaviour, IVolume
     {
         [SerializeField, FormerlySerializedAs("isGlobal")]
         private bool m_IsGlobal = true;
 
         /// <summary>
         /// Specifies whether to apply the Volume to the entire Scene or not.
         /// </summary>
-        [Tooltip("When enabled, the Volume is applied to the entire Scene.")]
```

Comment:
why is that tooltip removed?

Reply:
A tooltip on a property have no usage AFAIK. So it is dead code at the moment.
And in the Inspector the Volume have the Mode field to set it as global or not and it use a different tooltip than this one.

Reply:
ok thanks!